operators/project-quay/3.7.0-rc.2/quay-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,285 @@
+      mediatype: image/png
+  maturity: stable
+  links:
+    - name: Source Code
+      url: https://github.com/quay/quay-operator
+  maintainers:
+    - email: quay-sig@googlegroups.com
+      name: Project Quay Contributors
+  provider:
+    name: Red Hat
+  selector:
+    matchLabels:
+      alm-owner-quay-operator: quay-operator
+      operated-by: quay-operator
+  version: 3.7.0-rc.2
+  replaces: 3.7.0-rc.1
```

Comment:
Value for replaces should be `quay-operator.v3.7.0-rc.1`

Reply:
@HammerMeetNail ^

Reply:
Thanks! I've pushed the change and that test is passing now. Fingers crossed the rest of the tests pass and we an merge this in today :)